ui/index.html

```diff
@@ -1,39 +1,61 @@
 <!DOCTYPE html>
 <html>
-   <head>
-      <script src="jquery.js"></script>
-      <script src="rla_ui.js"></script>
-      <link rel="stylesheet" href="style.css">
-   </head>
-   <body>
-      <div id="auditTypeContainer" class="container">
-         Choose the audit type:<br />
-      </div>
-      <div id="auditNameContainer" class="container" style="display:none">
-         Enter a name for this audit:<br />
-         <input type="text" id="auditNameBox" />
-         <button id="auditNameSaveButton">Save</button>
-      </div>
-      <div id="cvrFileUploadContainer" class="container" style="display:none">
-         <p>Upload a Cast Vote Record (CVR) file</p>
-         <form id="uploadCVRForm" method="post" enctype="multipart/form-data">
+
+<head>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/js/bootstrap.min.js"></script>
+    <script src="rla_ui.js"></script>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <div id="auditTypeContainer" class="container">
+        Choose the audit type:
+        <br />
+    </div>
+    <div id="auditNameContainer" class="container" style="display:none">
+        Enter a name for this audit:
+        <br />
+        <!-- Regular expression to restrict user input. We call on the limitKeypress function which limits the amount of characters in the input field which is defined by maxlength. -->
+        <input type="text" id="auditNameBox" onkeypress="return event.charCode === 0 || /[a-zA-Z\-\_\d\.]+/.test(String.fromCharCode(event.charCode));" maxlength="15" onkeypress="limitKeypress(event,this.value,maxlength)" / />
```

Comment:
Can this `onkeypress` and the `onkeypress` below be pulled out into functions in `ui/rla_ui.js`?

Reply:
I made a function in Javascript and passed it to the seed and audit name functions.